src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -81,6 +81,27 @@ public async Task ConfigureAsync(CommandSettings command)
             _term.WriteLine("--------------------------------------------------------------------------------");
 
             Trace.Info(nameof(ConfigureAsync));
+
+
+            if (command.GenerateServiceConfig)
+            {
+#if OS_LINUX
+                if (!IsConfigured())
+                {
+                    throw new InvalidOperationException("--generateServiceConfig requires that the runner is already configured. For configuring a new runner as a service, run './config.sh'.");
+                }
+                // generate service config script for OSX and Linux, GenerateScripts() will no-opt on windows.
```

Comment:
we might want to add some trace to indicate we landed here.
`Trace.Info($"generate service config for runner: {setting.RunnerId}");`

☝️ or something like it. 😄

Reply:
Added a trace before. Do you typically trace a "completed" message after as well?